tests/app/celery/test_letters_pdf_tasks.py

```diff
@@ -56,188 +54,114 @@ def test_should_have_decorated_tasks_functions():
+    mocker.patch('app.celery.letters_pdf_tasks.get_letter_pdf_filename', return_value='LETTER.PDF')
     mock_retry = mocker.patch('app.celery.letters_pdf_tasks.create_letters_pdf.retry')
 
     create_letters_pdf(sample_letter_notification.id)
 
-    assert mock_get_letters_pdf.called
-    assert mock_retry.called
-
-
-def test_create_letters_pdf_handles_s3_errors(mocker, sample_letter_notification):
-    mocker.patch('app.celery.letters_pdf_tasks.get_letters_pdf', return_value=(b'\x00\x01', 1))
-    error_response = {
-        'Error': {
-            'Code': 'InvalidParameterValue',
-            'Message': 'some error message from amazon',
-            'Type': 'Sender'
-        }
-    }
-    mock_s3 = mocker.patch('app.letters.utils.s3upload', side_effect=ClientError(error_response, 'operation_name'))
-    mock_retry = mocker.patch('app.celery.letters_pdf_tasks.create_letters_pdf.retry')
-
-    create_letters_pdf(sample_letter_notification.id)
-
-    assert mock_s3.called
+    assert mock_celery.called
     assert mock_retry.called
 
 
 def test_create_letters_pdf_sets_technical_failure_max_retries(mocker, sample_letter_notification):
-    mock_get_letters_pdf = mocker.patch('app.celery.letters_pdf_tasks.get_letters_pdf', side_effect=RequestException)
+    mock_celery = mocker.patch('app.celery.letters_pdf_tasks.notify_celery.send_task', side_effect=Exception())
+    mocker.patch('app.celery.letters_pdf_tasks.get_letter_pdf_filename', return_value='LETTER.PDF')
     mock_retry = mocker.patch(
         'app.celery.letters_pdf_tasks.create_letters_pdf.retry', side_effect=MaxRetriesExceededError)
     mock_update_noti = mocker.patch('app.celery.letters_pdf_tasks.update_notification_status_by_id')
 
-    create_letters_pdf(sample_letter_notification.id)
+    with pytest.raises(NotificationTechnicalFailureException):
+        create_letters_pdf(sample_letter_notification.id)
 
-    assert mock_get_letters_pdf.called
+    assert mock_celery.called
     assert mock_retry.called
     mock_update_noti.assert_called_once_with(sample_letter_notification.id, 'technical-failure')
 
 
-def test_create_letters_gets_the_right_logo_when_service_has_no_logo(
-        notify_api, mocker, sample_letter_notification
-):
-    mock_get_letters_pdf = mocker.patch('app.celery.letters_pdf_tasks.get_letters_pdf', return_value=(b'\x00\x01', 1))
-    mocker.patch('app.letters.utils.s3upload')
-    mocker.patch('app.celery.letters_pdf_tasks.update_notification_status_by_id')
-
-    create_letters_pdf(sample_letter_notification.id)
-    mock_get_letters_pdf.assert_called_once_with(
-        sample_letter_notification.template,
-        contact_block=sample_letter_notification.reply_to_text,
-        filename=None,
-        values=sample_letter_notification.personalisation
-    )
-
-
 # We only need this while we are migrating to the new letter_branding model
 def test_create_letters_gets_the_right_logo_when_service_has_letter_branding_logo(
```

Comment:
Is this different to test_create_letters_pdf_happy_path?

Reply:
The only difference is that branding is set for a service and as a result there is a logo filename for the letter. I could do a parametrize for it if you think it's too repetitive ☺️